src/site/content/en/blog/interop-2022/index.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: "Interop 2022: browsers working together to improve the web for developers"
+subhead:
+    "For the first time ever, all major browser vendors, and other stakeholders, have come together to solve the top browsers compatibility issues identified by web developers. Interop 2022 will improve the experience of developing for the web in 15 key areas. In this article, find out how we got here, what the project focuses on, how success will be measured, and how you can track progress."
+description:
+    "Learn more about how all browser vendors, and other stakeholders, have come together to solve the top browsers compatibility issues identified by web developers. Interop 2022 will improve the experience of developing for the web in 15 key areas."
+authors:
+  - rachelandrew
+  - robertnyman
+  - foolip
+hero: "image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/xv75zUKa4jUR2xGMZ7Xb.jpg"
+alt: "A picture of a brick wall."
+date: 2022-03-03
+tags:
+  - blog
+  - css
+---
+
+For the first time ever, all major browser vendors, and other stakeholders, have come together to solve the top browsers compatibility issues identified by web developers. Interop 2022 will improve the experience of developing for the web in 15 key areas. In this article, find out how we got here, what the project focuses on, how success will be measured, and how you can track progress.
```

Comment:
We probably want to remove this because it's repetitive of the subhead (or change the subhead).

Reply:
Absolutely. I didn't know if the subhead will be visible in the post or not, hence the repetition.